### ISSUE TITLE
ChromaSpectrogram is different with librosa

### ISSUE THREAD

**Issue by 980202006** [pytorch/audio#3889]
### 🐛 Describe the bug

Compared to librosa's ChromaSpectrogram, torchaudio's ChromaSpectrogram misses some notes.
`
import numpy as np
import matplotlib.pyplot as plt
import librosa
model = nn.Sequential( transforms.ChromaSpectrogram(
            sample_rate=44100,n_fft=2048, n_chroma=12),
            nn.Softmax(-1),).cuda()
outs = model(x) 
fig, ax = plt.subplots()
img = librosa.display.specshow(outs[0][:, :600].detach().cpu().numpy(), ax=ax)
fig.colorbar(img, ax=ax)
fig.savefig('torchaudio.png')
outs = librosa.feature.chroma_cqt(y=x.cpu().numpy(), sr=44100,hop_length=512, n_fft=2048)
fig, ax = plt.subplots()
img = librosa.display.specshow(outs[0][:, :600].detach().cpu().numpy(), ax=ax)
fig.colorbar(img, ax=ax)
fig.savefig('torchaudio.png')
`
This is librosa's chroma.
![Image](https://github.com/user-attachments/assets/bf04d42d-3ccb-4438-8aee-20bb4d6eae63)
This is torchaudio's chroma.
![Image](https://github.com/user-attachments/assets/3fa2d42a-5a8c-4964-8966-4140872f67e3)

### Versions

torchaudio: '2.5.1+cu124'
librosa: '0.10.1'
